### PR TITLE
[EN/ZH] TNMT: Initial article

### PR DESCRIPTION
**Pull request by kj415j45**

### CODE REVIEW

File: wiki/Tournaments/TNMT/1/en.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+![TNMT Logo](img/logo.png)
+
+The **:thonking: Newbie mania Tournament** (***TNMT***) is a one-on-one osu!mania 4K tournament hosted by CJB and TMT Team.
+
+## Tournament Schedule
+
+| Event | Time |
+| --: | :-- |
+| Registration Phase | 2019-07-13/2019-08-03 |
+| Qualifiers Showcase | 2019-8-10 |
```
Comment (fajar13k) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
| Qualifiers Showcase | 2019-08-10 |
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+## Prize
+
+All prizes are sponsored by **![][flag_CN] [hans1999](https://osu.ppy.sh/users/6679329)**.
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![](/wiki/shared/GCrown.png) | 3 months of osu!supporter tag, unique profile badge |
+| ![](/wiki/shared/SCrown.png) | 2 months of osu!supporter tag |
+| ![](/wiki/shared/BCrown.png) | 1 months of osu!supporter tag |
```
Comment (fajar13k) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
| ![](/wiki/shared/BCrown.png) | 1 month of osu!supporter tag |
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+| Designer | ![][flag_HK] [Chaoslitz](https://osu.ppy.sh/users/3621552) |
+| Wiki | ![][flag_CN] [kj415j45](https://osu.ppy.sh/users/9367540) |
+
+## Links
+
+- [Sign up](https://docs.google.com/forms/d/e/1FAIpQLSfq_yw61wfi9NyLxmTvBDw-rzwyqsYYHt_5udh5pMmX0iPQdA/viewform)
+- [Challonge](https://challonge.com/TNMT)
+- [Discord](https://discord.gg/nxWEjpW)
+- [QQ Group](https://jq.qq.com/?_wv=1027&k=5XdDR3K)
+- [Forum post](https://osu.ppy.sh/community/forums/topics/933542)
```
Comment (fajar13k) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
- [Discussion thread](https://osu.ppy.sh/community/forums/topics/933542)
```
And you may want to put it on the top

File: wiki/Tournaments/TNMT/1/en.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+| Commentators | ![][flag_CN] [crazybrother](https://osu.ppy.sh/users/8045083), ![][flag_CN] [hans1999](https://osu.ppy.sh/users/6679329), ![][flag_CN] [NimaPro](https://osu.ppy.sh/users/11524571), ![][flag_SG] [Polytetral](https://osu.ppy.sh/users/8612061), ![][flag_CN] [Rasis](https://osu.ppy.sh/users/1209363), ![][flag_CN] [Snowkyou](https://osu.ppy.sh/users/12513448), ![][flag_AU] [PotassiumF](https://osu.ppy.sh/users/4247722) |
+| Designer | ![][flag_HK] [Chaoslitz](https://osu.ppy.sh/users/3621552) |
+| Wiki | ![][flag_CN] [kj415j45](https://osu.ppy.sh/users/9367540) |
+
+## Links
+
+- [Sign up](https://docs.google.com/forms/d/e/1FAIpQLSfq_yw61wfi9NyLxmTvBDw-rzwyqsYYHt_5udh5pMmX0iPQdA/viewform)
+- [Challonge](https://challonge.com/TNMT)
```
Comment (fajar13k) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
- [Challonge brackets](https://challonge.com/TNMT)
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+| Referees | ![][flag_CN] [Flow Light](https://osu.ppy.sh/users/6705256), ![][flag_HK] [zero2snow](https://osu.ppy.sh/users/7751516), ![][flag_CN] [kj415j45](https://osu.ppy.sh/users/9367540), ![][flag_CN] [Rasis](https://osu.ppy.sh/users/1209363), ![][flag_CN] [IA daisuki](https://osu.ppy.sh/users/11238501), ![][flag_CN] [crazybrother](https://osu.ppy.sh/users/8045083), ![][flag_CN] [aiyulu](https://osu.ppy.sh/users/189617), ![][flag_CA] [Sinaeb](https://osu.ppy.sh/users/1576095) |
+| Commentators | ![][flag_CN] [crazybrother](https://osu.ppy.sh/users/8045083), ![][flag_CN] [hans1999](https://osu.ppy.sh/users/6679329), ![][flag_CN] [NimaPro](https://osu.ppy.sh/users/11524571), ![][flag_SG] [Polytetral](https://osu.ppy.sh/users/8612061), ![][flag_CN] [Rasis](https://osu.ppy.sh/users/1209363), ![][flag_CN] [Snowkyou](https://osu.ppy.sh/users/12513448), ![][flag_AU] [PotassiumF](https://osu.ppy.sh/users/4247722) |
+| Designer | ![][flag_HK] [Chaoslitz](https://osu.ppy.sh/users/3621552) |
+| Wiki | ![][flag_CN] [kj415j45](https://osu.ppy.sh/users/9367540) |
+
+## Links
+
+- [Sign up](https://docs.google.com/forms/d/e/1FAIpQLSfq_yw61wfi9NyLxmTvBDw-rzwyqsYYHt_5udh5pMmX0iPQdA/viewform)
```
Comment (fajar13k) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
- [Registration form](https://docs.google.com/forms/d/e/1FAIpQLSfq_yw61wfi9NyLxmTvBDw-rzwyqsYYHt_5udh5pMmX0iPQdA/viewform)
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+- [Challonge](https://challonge.com/TNMT)
+- [Discord](https://discord.gg/nxWEjpW)
```
Comment (fajar13k) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
- [Discord server](https://discord.gg/nxWEjpW)
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
+# :thonking: Newbie mania Tournament
+
+![TNMT Logo](img/logo.png)
+
+The **:thonking: Newbie mania Tournament** (***TNMT***) is a one-on-one osu!mania 4K tournament hosted by CJB and TMT Team.
+
+## Tournament Schedule
+
+| Event | Time |
```
Comment (fajar13k) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
| Event | Timestamp |
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+  - [Sinaeb](https://www.twitch.tv/sinbae_)
+
+## Ruleset
+
+### Registration
+
+- The following conditions should be satisfied to register in the Tournaments.
+  - Rank of osu!mania should **not above #4000**, and the highest historical rank is **lower than #3000**.
+  - Have a good standing in osu!community and other tournaments in recent years.
+- List of participants will be submit to osu! Tournament Team. Player who don't meet the requirements will be removed from list.
```
Comment (fajar13k) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
- List of participants will be submitted to the osu! Tournament Team. Player who do not meet the requirements will be removed from the list.
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+- Staff (except commentators) are not allowed to sign up.
+- After osu! Tournament Team's review, players met requirements will receive notification email.
```
Comment (fajar13k) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
- After osu! Tournament Team's review, players whom met requirements will receive notification via email.
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+
+### Overview
+
+All the mappools are in FreeMod. If player want to enable DT, NC or HT, the opponent must agree with it and inform the referee. All mappools except Qualifiers have a Tiebreaker.
```
Comment (fajar13k) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
All the mappools are in FreeMod. If player wants to enable DT, NC or HT, both players must agree with the decision and inform the referee about the changes. All mappools except Qualifiers have a Tiebreaker.
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+  - [IA daisuki](https://live.bilibili.com/4353181)
+  - [kj415j45](https://okinawa.loli.land/live)
+  - [Rasis](https://live.bilibili.com/5333)
+  - [Sinaeb](https://www.twitch.tv/sinbae_)
+
+## Ruleset
+
+### Registration
+
+- The following conditions should be satisfied to register in the Tournaments.
```
Comment (fajar13k) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
- Participants must met the following conditions in order for their registration to count.
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+  - Players should enter the room 10 minutes before match time.
+- Disconnection
+  - If disconnect happened **in 30 seconds after map started**
+    - Inform the referee so disconnected player can ask for re-attempted.
+    - Only 2 chances to re-attempted.
+  - If disconnect happened **in 30 seconds before map end**
+    - If player finish the map, he should upload the score screenshot through `Shift`+`F12` and give the link to referee.
+- Match (not apply to Qualifiers)
+  - Room settings will be **Team VS, Score V2**.
+  1. Warm-up
```
Comment (fajar13k) [inline wiki/Tournaments/TNMT/1/en.md]: Line the indentation, it's currently broken (for others too)

File: wiki/Tournaments/TNMT/1/en.md [hunk 13]
```diff
@@ -0,0 +1,136 @@
...
+
+### Qualifiers
+
+- The Team will release a schedule of Qualifiers, players can select which time to play from it. However, Max player of each time is 6. First come, first served.
+- Room settings will be **Head To Head, Score V2**.
+- Referee will select map as the order of pool and each map will be played twice. The highest score will be recorded. Only **1 minute** between each play.
+
+### Group Stage
+
+- Group Stage will play in single round-robin. The best **2 players** of each group will advance to Round of 16. It arranged from highest priority to lowest:
```
Comment (fajar13k) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
- Group Stage will be played under single round-robin. **2 best players** from each group will advance to Round of 16 stage. The priority range from highest to lowest priority explained as follows:
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 14]
```diff
@@ -0,0 +1,136 @@
...
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![](/wiki/shared/GCrown.png) | 3 months of osu!supporter tag, unique profile badge |
+| ![](/wiki/shared/SCrown.png) | 2 months of osu!supporter tag |
+| ![](/wiki/shared/BCrown.png) | 1 months of osu!supporter tag |
+
+## Organisation
+
+| Position | Member(s) |
+| :-: | :-- |
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
| :-- | :-- |
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 15]
```diff
@@ -0,0 +1,136 @@
...
+| Management | ![][flag_CN] [hans1999](https://osu.ppy.sh/users/6679329), ![][flag_CA] [Sinaeb](https://osu.ppy.sh/users/1576095) |
+| Coordinator | ![][flag_SG] [Polytetral](https://osu.ppy.sh/users/8612061), ![][flag_CN] [kj415j45](https://osu.ppy.sh/users/9367540) |
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
| Coordinators | ![][flag_SG] [Polytetral](https://osu.ppy.sh/users/8612061), ![][flag_CN] [kj415j45](https://osu.ppy.sh/users/9367540) |
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 16]
```diff
@@ -0,0 +1,136 @@
...
+      - Should not pick the Tiebreaker.
+      - The winner of rolling points will pick the first map, then exchange host every round.
+      - The interval between rounds shall **not be longer than 3 minutes**. Referee has right to force-start the round in 15 seconds after that.
+      - If draw, this round will be treated as invalid and the map will be marked as un-picked. Exchange the picker and play other map.
+  4. Tiebreaker
+      - If both players reach the match point, the Tiebreaker will be used in this round.
+  5. Pause
+      - If player has some unexpected things to do and need a pause, inform the referee and set a timer. Pause can **not be longer than 5 minutes**.
+- Overtime
+  - Player not enter his room in time, or disconnect from a started match will be treated as Overtime. If Overtime timer of the player reached **10 minutes**, he will lost the match.
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
  - Players who do not enter his room in time, or disconnect from a started match will be treated as Overtime. If the Overtime timer of the player reached **10 minutes**, they will lose the match.
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 17]
```diff
@@ -0,0 +1,136 @@
...
+### Qualifiers
+
+- The Team will release a schedule of Qualifiers, players can select which time to play from it. However, Max player of each time is 6. First come, first served.
+- Room settings will be **Head To Head, Score V2**.
+- Referee will select map as the order of pool and each map will be played twice. The highest score will be recorded. Only **1 minute** between each play.
+
+### Group Stage
+
+- Group Stage will be played under single round-robin. **2 best players** from each group will advance to Round of 16 stage. The priority range from highest to lowest priority explained as follows:
+  1. More matches win.
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
  1. More matches won.
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 18]
```diff
@@ -0,0 +1,136 @@
...
+
+- Join match
+  - The match room has **TNMT** as prefix and players in this match will receive invitation.
+  - Players should enter the room 10 minutes before match time.
+- Disconnection
+  - If disconnect happened **in 30 seconds after map started**
+    - Inform the referee so disconnected player can ask for re-attempted.
+    - Only 2 chances to re-attempted.
+  - If disconnect happened **in 30 seconds before map end**
+    - If player finish the map, he should upload the score screenshot through `Shift`+`F12` and give the link to referee.
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
    - If the player finishes the map, they should upload the score screenshot through `Shift`+`F12` and give the link to referee.
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 19]
```diff
@@ -0,0 +1,136 @@
...
+  - [kj415j45](https://okinawa.loli.land/live)
+  - [Rasis](https://live.bilibili.com/5333)
+  - [Sinaeb](https://www.twitch.tv/sinbae_)
+
+## Ruleset
+
+### Registration
+
+- Participants must met the following conditions in order for their registration to count.
+  - Rank of osu!mania should **not above #4000**, and the highest historical rank is **lower than #3000**.
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
  - Rank of osu!mania should **not be above #4000**, and the highest historical rank is **lower than #3000**.
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 20]
```diff
@@ -0,0 +1,136 @@
...
+  - Have a good standing in osu!community and other tournaments in recent years.
+- List of participants will be submitted to the osu! Tournament Team. Player who do not meet the requirements will be removed from the list.
+- Staff (except commentators) are not allowed to sign up.
+- After osu! Tournament Team's review, players whom met requirements will receive notification via email.
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
- After osu! Tournament Team's review, players whom met requirements will receive a notification via email.
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 21]
```diff
@@ -0,0 +1,136 @@
...
+
+### Overview
+
+All the mappools are in FreeMod. If player wants to enable DT, NC or HT, both players must agree with the decision and inform the referee about the changes. All mappools except Qualifiers have a Tiebreaker.
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
All the mappools are in FreeMod. If a player wants to enable DT, NC or HT, both players must agree with the decision and inform the referee about the changes. All mappools except Qualifiers have a Tiebreaker.
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 22]
```diff
@@ -0,0 +1,136 @@
...
+      - Should not pick the map that banned in Roll & Ban.
+      - Should not pick the map that picked.
+      - Should not pick the Tiebreaker.
+      - The winner of rolling points will pick the first map, then exchange host every round.
+      - The interval between rounds shall **not be longer than 3 minutes**. Referee has right to force-start the round in 15 seconds after that.
+      - If draw, this round will be treated as invalid and the map will be marked as un-picked. Exchange the picker and play other map.
+  4. Tiebreaker
+      - If both players reach the match point, the Tiebreaker will be used in this round.
+  5. Pause
+      - If player has some unexpected things to do and need a pause, inform the referee and set a timer. Pause can **not be longer than 5 minutes**.
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
      - If the player has some unexpected things to do and needs a pause, they should inform the referee and a timer will be set. Pause can **not be longer than 5 minutes**.
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 23]
```diff
@@ -0,0 +1,136 @@
...
+  - Have a good standing in osu!community and other tournaments in recent years.
+- List of participants will be submitted to the osu! Tournament Team. Player who do not meet the requirements will be removed from the list.
+- Staff (except commentators) are not allowed to sign up.
+- After osu! Tournament Team's review, players whom met requirements will receive notification via email.
+
+### Overview
+
+All the mappools are in FreeMod. If player wants to enable DT, NC or HT, both players must agree with the decision and inform the referee about the changes. All mappools except Qualifiers have a Tiebreaker.
+
+- Qualifiers: **32 leading players** descending order by total score will advance to Group Stage. **4 maps** in pool.
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
- Qualifiers: **32 leading players** in descending order by total score will advance to Group Stage. **4 maps** in pool.
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 24]
```diff
@@ -0,0 +1,136 @@
...
+
+CJB & TMT Team has the ultimate power of interpretation.
+
+- **Notice: Any and all whom we deem as a "cheater" or "suspicious", screened by both the tourney and the official osu! staff, will be removed from tournament immediately.**
+
+- Join match
+  - The match room has **TNMT** as prefix and players in this match will receive invitation.
+  - Players should enter the room 10 minutes before match time.
+- Disconnection
+  - If disconnect happened **in 30 seconds after map started**
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
  - If disconnect happened **30 seconds after map started**
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 25]
```diff
@@ -0,0 +1,136 @@
...
+    - Inform the referee so disconnected player can ask for re-attempted.
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
    - Inform the referee so disconnected player can ask to reattempt.
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 26]
```diff
@@ -0,0 +1,136 @@
...
+    - Only 2 chances to re-attempted.
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
    - Only 2 chances to reattempt.
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 27]
```diff
@@ -0,0 +1,136 @@
...
+  - If disconnect happened **in 30 seconds before map end**
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
  - If disconnect happened **30 seconds before map ends**
```

File: wiki/Tournaments/TNMT/1/zh.md [hunk 28]
```diff
@@ -0,0 +1,136 @@
...
+
+:thonking: Newbie mania Tournament（以下均简写为 TNMT 4K 2019）为 osu!mania 4K 个人比赛，是由 CJB 和 TMT 团队联合举办的第一届国际 osu!mania 4K 大赛。
+
+## 比赛时间表
+
+| 阶段 | 时间 |
+| --: | :-- |
+| 报名 | 2019-07-13/2019-08-03 |
+| 资格赛图池展示 | 2019-08-10 12:00(UTC+0) |
+| 资格赛 | 2019-0817/2019-08-18 |
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/zh.md]:
```suggestion
| 资格赛 | 2019-08-17/2019-08-18 |
```

File: wiki/Tournaments/TNMT/1/zh.md [hunk 29]
```diff
@@ -0,0 +1,136 @@
...
+| 小组赛抽签仪式 | 2019-08-18 12:00(UTC+0) |
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/zh.md]:
```suggestion
| 小组赛抽签仪式 | 2019-08-18 (12:00 UTC+0) |
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 30]
```diff
@@ -0,0 +1,136 @@
...
+The **:thonking: Newbie mania Tournament** (***TNMT***) is a one-on-one osu!mania 4K tournament hosted by CJB and TMT Team.
+
+## Tournament Schedule
+
+| Event | Timestamp  |
+| --: | :-- |
+| Registration Phase | 2019-07-13/2019-08-03 |
+| Qualifiers Showcase | 2019-08-10 |
+| Qualifiers | 2019-08-17/2019-08-18 |
+| Live Drawings | 2019-08-18 |
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]: Taken from translation.
```suggestion
| Live Drawings | 2019-08-18 (12:00 UTC+0) |
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 31]
```diff
@@ -0,0 +1,136 @@
...
+![TNMT Logo](img/logo.png)
+
+The **:thonking: Newbie mania Tournament** (***TNMT***) is a one-on-one osu!mania 4K tournament hosted by CJB and TMT Team.
+
+## Tournament Schedule
+
+| Event | Timestamp  |
+| --: | :-- |
+| Registration Phase | 2019-07-13/2019-08-03 |
+| Qualifiers Showcase | 2019-08-10 |
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]: Taken from translation.
```suggestion
| Qualifiers Showcase | 2019-08-10 (12:00 UTC+0) |
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 32]
```diff
@@ -0,0 +1,136 @@
...
+  - [Sinaeb](https://www.twitch.tv/sinbae_)
+
+## Ruleset
+
+### Registration
+
+- Participants must met the following conditions in order for their registration to count.
+  - Rank of osu!mania should **not above #4000**, and the highest historical rank is **lower than #3000**.
+  - Have a good standing in osu!community and other tournaments in recent years.
+- List of participants will be submitted to the osu! Tournament Team. Player who do not meet the requirements will be removed from the list.
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
- List of participants will be submitted to the osu! Tournament Team. Players who do not meet the requirements will be removed from the list.
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 33]
```diff
@@ -0,0 +1,136 @@
...
+  - Finals - Week 2: **15+1 maps** in pool, **BO13**.
+
+### General Rules
+
+CJB & TMT Team has the ultimate power of interpretation.
+
+- **Notice: Any and all whom we deem as a "cheater" or "suspicious", screened by both the tourney and the official osu! staff, will be removed from tournament immediately.**
+
+- Join match
+  - The match room has **TNMT** as prefix and players in this match will receive invitation.
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
  - The match room has **TNMT** as prefix and players in this match will receive an invitation.
```

**Comment by Joehuu** [comment]
should've made two commits with my reviews and new changes. u also didn't use github web interface's batch suggestions to commit so i have to check it again

File: wiki/Tournaments/TNMT/1/en.md [hunk 34]
```diff
@@ -0,0 +1,140 @@
...
+![TNMT Logo](img/logo.png)
+
+The **:thonking: Newbie mania Tournament** (***TNMT***) is a one-on-one osu!mania 4K tournament hosted by CJB and TMT Team.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2019-07-13/2019-08-03 |
+| Qualifiers Showcase | 2019-08-10 (12:00 UTC+0)  |
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
| Qualifiers Showcase | 2019-08-10 (12:00 UTC+0) |
```

File: wiki/Tournaments/TNMT/1/en.md [hunk 35]
```diff
@@ -0,0 +1,140 @@
...
+| Qualifiers | 2019-08-17/2019-08-18 |
+| Live Drawings | 2019-08-18 (12:00 UTC+0)  |
```
Comment (Joehuu) [inline wiki/Tournaments/TNMT/1/en.md]:
```suggestion
| Live Drawings | 2019-08-18 (12:00 UTC+0) |
```

**Comment by kj415j45** [comment]
actually, I apply all of your suggestions :p

**Comment by kj415j45** [comment]
@Joehuu, if the author commits from cli, the suggestions won't be folded even though they were marked as outdated. The author should manually click "Resolve conversation" to fold the suggestions, which means the suggestions were applied.

**Pull request merged**